Makefile

```diff
@@ -42,40 +42,55 @@
 #
 #
 ###############################################################################
-# The build architecture is select by setting the ARCH variable.
-# For example: When building on ppc64le you could use ARCH=ppc64le make <....>.
-# When ARCH is undefined it defaults to amd64.
-ARCH?=amd64
-ifeq ($(ARCH),amd64)
-	ARCHTAG?=
-	GO_BUILD_VER?=v0.9
-	PROTOC_VER?=v0.1
-	FV_TYPHAIMAGE?=calico/typha:v0.6.0-beta1-16-g512a0f2
-endif
+# Both native and cross architecture builds are supported.
```

Comment:
Please can you list the known-supported ARCHes?

Reply:
Basically, any one in `docker-images/Dockerfile.*`. I will use the code from the `go-build` Makefile to list them out.

Reply:
Done. It adds it to the output of `make help`.

Reply:
Actually, I added a little more instructional help there.